buildSrc/src/main/java/Config.kt

```diff
@@ -46,4 +51,14 @@ object Config {
         val tvosDeploymentTarget = "11.0"
         val watchosDeploymentTarget = "4.0"
     }
+
+    object Sentry {
+        val kotlinMultiplatformSdkName = "sentry.kotlin.kotlin-multiplatform"
```

Comment:
Not sure if this is correct.
I'd double check our docs about the SDK naming

Reply:
Another alternative could be `sentry.java.android.kotlin-multiplatform` or `sentry.cocoa.kotlin-multiplatform` depending on which platform the user uses. I guess this is more fitting since we are using multiple SDKs and the docs specify it using unity as an example

Reply:
I guess I mistakenly got confused with the dot and dash.
Maybe `sentry.kotlin.kmp?`